variables.tf

```
@@ -130,3 +130,15 @@ variable "login_service_accounts" {
   type        = list(string)
   default     = []
 }
+
+variable "fw_allowed_cidrs" {
+  description = "List of source IP address of particular range to which the firewall rule will apply. These ranges must be expressed in CIDR format."
```

These are not `IP address`. These are IP CIDRs.

```suggestion
  description = "List of IP CIDRs to be allowed  by the VM firewall rules for incoming traffic."
```